.dockerignore

```diff
@@ -0,0 +1,24 @@
+.github
```

Comment:
I like the idea of using a dockerignore file to possibly reduce the image size 👍

Reply:
Actually that doesn't reduce the image size since the local directory is copied (with the `COPY` statement) only in the build stage.

Reply:
However that helps caching and preventing useless re-build of images. It's also considered a good practice.

Reply:
Ah ok I see.  Good to know, thanks!